manager/integration/tests/test_basic.py

```diff
@@ -4588,21 +4593,34 @@ def test_backup_volume_restore_with_access_mode(set_random_backupstore, # NOQA
        => Validate the access mode should be the same as the specified
     """
     # Step 1
-    test_volume_name = generate_volume_name()
+    test_volume_name = "test-volume-" + str(random.randint(0, 9))
     client.create_volume(name=test_volume_name,
                          size=str(DEFAULT_VOLUME_SIZE * Gi),
                          numberOfReplicas=2,
                          accessMode=access_mode)
     wait_for_volume_creation(client, test_volume_name)
     volume = wait_for_volume_detached(client, test_volume_name)
-    volume.attach(hostId=common.get_self_host_id())
-    volume = common.wait_for_volume_healthy(client, test_volume_name)
+
+    # Since RWX volumes can only be manually attached in maintenance mode,
+    #  the volume was created by PV and PVC deployed.
+    if access_mode == "rwx":
+        test_pv_name = test_volume_name + "-pv"
+        test_pvc_name = test_volume_name + "-pvc"
```

Comment:
Suggest referring to other cases by using `pv_name = get_volume_name(core_api, pvc_name)` for using the similar convention.

Reply:
Since the `create_pvc_for_volume()` need a PV which was already exists or it will return error message. 

Therefore, I think pv_name can't be created through pvc_name, but individually constructed PVC and PV

Reply:
It's not right. PV is created after the PVC.

Reply:
oh, you are talking about the function calls instead. Yeah, then it's OK here!